### ISSUE TITLE
Possible to play dance by music with custom settings?

### ISSUE THREAD

**Issue by HaKaShis** [PavelDoGreat/WebGL-Fluid-Simulation#10]
HI,

Love what you have done with this ! 
I would love if it could dance to music and use a projector on the wall for this beautiful display, with same settings and maybe a random setting pattern with a timer.
Not a coder myself unfortunately...

thanks for the beautiful simulation !

**Comment by type76** [comment]
I need this in my life. What a great idea.
I might be able to edit my branch to get what you want, but someone else can do better.
https://github.com/type76/fluid

**Comment by HaKaShis** [comment]
> I need this in my life. What a great idea.
> I might be able to edit my branch to get what you want, but someone else can do better.
> https://github.com/type76/fluid
> 
> update:
> https://type76.com/fluid/audio1.html
Great work, unfortunately the demo does not play as the wet.mp3 file is not on your host folder, I downloaded the source added my own .mp3 it does play now but does not react to it. Is it even possible to react to PC music output ? Alternative would be pick up mic and react to it.

**Comment by type76** [comment]
Yes, sorry.. forgot to mention that the mp3 isn't linked and the input isn't working either. The general idea I was going for is to load the mp3, analyse it (in real time) and play splats or multi plats with the filters. I'm still trying to link all the bits up though. Then the fun can begin.

This is still broken:
https://github.com/type76/fluid/blob/master/audioscript.js

It can be seen here:
https://type76.com/fluid/a.html

**Comment by type76** [comment]
It lives..
https://type76.com/fluid/v2.html
